### DISCUSSION TITLE
Codespace creation failed

### DISCUSSION THREAD

**Discussion by johnpisera** [community/community#34618] [Codespaces]
Hello all,

I am currently enrolled in cs50 and ever since the outage yesterday (9/28/22) I have been unable to access my codespace. I see both on github and on cs50 status pages that all systems are operational. I have restarted my computer and cleared my browser but I'm still  either receiving messages that say "Codespace creation failed" or  a different message that mentions something about my "codespacestatewatcher"? I am able to log in on my mac laptop however I prefer to work on my desktop running windows 10 where I am receiving errors.

Any advice would be greatly appreciated!

-John P.

**Comment by softwaredevelop** [comment]
Hello

Try VS Code with GitHub [Codespaces extension](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces)

Good luck

**Reply by johnpisera** [reply]
Thank you but unfortunately that isn't loading my codespace either. It tries to fetch the data a few times before I receive a 404 error.

**Reply by softwaredevelop** [reply]
I don't know, I found about this problem:
https://www.reddit.com/r/cs50/comments/u9acav/comment/i5vzstk/
https://www.reddit.com/r/cs50/comments/sdj2ix/codespace_does_not_work_at_all/

**Reply by johnpisera** [reply]
Thank you, the first link was helpful solving my problem. I deleted my codespaces and uninstalled then reinstalled google chrome and that worked. Careful not to delete any codespaces to anyone doing this method to resolve.
